PyRDF/backend/Backend.py

```diff
@@ -14,6 +15,30 @@ class Backend(ABC):
         backend.
 
     """
+    supported_operations = {
+        'Define':Operation.Types.TRANSFORMATION,
+        'Filter':Operation.Types.TRANSFORMATION,
+        'Histo1D':Operation.Types.ACTION,
+        'Histo2D':Operation.Types.ACTION,
+        'Histo3D':Operation.Types.ACTION,
+        'Profile1D':Operation.Types.ACTION,
+        'Profile2D':Operation.Types.ACTION,
+        'Profile3D':Operation.Types.ACTION,
+        'Count':Operation.Types.ACTION,
+        'Min':Operation.Types.ACTION,
+        'Max':Operation.Types.ACTION,
+        'Mean':Operation.Types.ACTION,
+        'Sum':Operation.Types.ACTION,
+        'Fill':Operation.Types.ACTION,
+        'Report':Operation.Types.ACTION
+        }
+
+    future_operations = [
```

Comment:
I think it is not even worth it, but I would let the others comment

Reply:
Ok @etejedor . I'll remove it for now then 👍

Reply:
I think `supported_operations` in `Backend` should contain absolutely everything, including what is now in `future_operations`. Then the rest of the backends can redefine those containers.

Reply:
@etejedor , I think it'd better if `Operation` class' `operations_dict` itself contains all the operations including the future ones. Because, the ones in `Operation` is immaterial of the backend right ?

Reply:
But @etejedor , what would be the operation type of Instant actions like `Snapshot` or `Foreach` ?

Reply:
For me they are both actions.

Reply:
@diocas @JavierCVilla @bluehood opinions here?

Reply:
Ok @etejedor . I'll keep them as actions for now 👍